frontend/src/my-books/MyBooks.tsx

```diff
@@ -26,15 +26,12 @@ interface IState {
 class MyBooks extends Component<Record<string, unknown>, IState> {
     render() {
         return (
-            <div>
-                <NavBar />
-                <div>
-                    <ShelfCarousel title="Reading" />
```

Comment:
Looks like we can bump the font-size up a tad for the ShelfCarousel titles to 24px based of specs in Figma :)